dora-metrics/pelorus
# fix: Restrict INFO log level verbosity

## Linked Issues

resolves #1006

## Description

Restricts the amount of logs when user sets it to INFO (the default).

## Testing Instructions

Check the logs of Committime, Deploytime, failure and webhook exporters with log level set to INFO.
